src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx

```diff
@@ -576,6 +576,27 @@ Here are some tips for creating and using a NRQL condition:
   </tbody>
 </table>
 
+## Edits can reset condition evaluation [#evaluation-resets]
+
+When you edit alert conditions in some specific ways (detailed below), their evaluations are reset, meaning that any evaluation up until that point is lost, and the evaluation starts over from that point. 
+
+For [baseline conditions](#threshold-types), this has the following effects:
```

Comment:
For all NRQL condition types, this can have either of the below effects. The first one is just for baselines, the 2nd one is for any condition that uses a buffer.